internal/learn/about-iota/coordinator.md

```diff
@@ -0,0 +1,23 @@
+---
+id: coordinator
+title: Coordinator
+description: What is Coordinator in the IOTA network.
+---
+
+# Coordinator
+
+:::note
+
+Coordinator is a temporary solution. IOTA will remove Coordinator in the future [IOTA 2.0 update](./roadmap-to-decentralization.md).
+
+:::
+
+Coordinator is a client that sends signed messages called milestones that nodes trust and use to confirm messages. Messages in the Tangle are considered for confirmation only when they are directly or indirectly referenced by a milestone that nodes have validated.
+
+To allow the nodes to recognize valid milestones, all nodes that participate in the same IOTA network are configured with the Ed25519 signatures of a coordinator they trust. By knowing those signatures, nodes can validate the signatures in issued milestones to verify whether the trusted coordinator did sign them.
```

Comment:
```suggestion
To allow the nodes to recognize valid milestones, all nodes participating in the same IOTA network are configured with the [Ed25519 signatures](https://wiki.iota.org/chrysalis-docs/introduction/what_is_chrysalis#ed25519-signature-scheme) of a coordinator they trust. By knowing those signatures, nodes can validate the signatures in issued milestones to verify whether the trusted coordinator did sign them.
```

Reply:
I've removed the mention of Ed25519. Same concern as with crysalis in general.